Poker.BE/Poker.BE.Domain/Game/Player.cs

```diff
@@ -13,24 +14,40 @@ public enum State
         {
             ActiveUnfolded,
             ActiveFolded,
+            ActiveAllIn,
             Passive
         }
 
         public static readonly int NPRIVATE_CARDS = 2;
         #endregion
 
         #region Properties
-        public State CurrentState { get; private set; }
+        public State CurrentState { get; set; }
         public Card[] PrivateCards { get; set; }
+        public Wallet Wallet { get; }
         #endregion
 
         #region Constructors
         public Player()
         {
             PrivateCards = new Card[NPRIVATE_CARDS];
             CurrentState = State.Passive;
+            Wallet = new Wallet(0);
         }
         #endregion
 
+        #region Methods
+        public void AddMoney(int amount)
+        {
+            this.Wallet.amountOfMoney += amount;
```

Comment:
OK, this function is good for now, but at my UCC i have done a very different implementation. we will need to figure it out at merge. luckily - we got a lot of tests! so it will make it easier to know when we created bugs during the process of merging. next time - try to use more stubs and mocks so we don't implement the same classes.